python/bindings.cpp

```diff
@@ -315,31 +325,15 @@ void bindReportReader(py::module& m, const std::string& prefix) {
     py::class_<typename ReportType::Population>(m,
                                                 (prefix + "ReportPopulation").c_str(),
                                                 "A population inside a ReportReader")
-        .def(
-            "get",
-            [](const typename ReportType::Population& self) { return self.get(); },
-            "Return all reports")
-        .def(
-            "get",
-            [](const typename ReportType::Population& self, double tstart, double tstop) {
-                return self.get(Selection({}), tstart, tstop);
-            },
-            "Return reports between 'tstart' and 'tstop'",
-            "tstart"_a,
-            "tstop"_a)
-        .def(
-            "get",
-            [](const typename ReportType::Population& self, Selection sel) {
-                return self.get(sel);
-            },
-            "Return reports with all those node_ids",
-            "node_ids"_a)
         .def("get",
              &ReportType::Population::get,
              "Return reports with all those node_ids between 'tstart' and 'tstop'",
-             "node_ids"_a,
-             "tstart"_a,
-             "tstop"_a)
+             "node_ids"_a = nonstd::nullopt,
+             "tstart"_a = nonstd::nullopt,
+             "tstop"_a = nonstd::nullopt)
+        .def("get_nodes_ids",
```

Comment:
`get_node_ids` with no `s` to nodes to be consistent with the SpikeReport / FrameReport etc ...